src/modules/dashboard/components/CreatePaymentDialog/CreatePaymentDialogForm.tsx

```diff
@@ -0,0 +1,386 @@
+              delimiter: ',',
+              numeral: true,
+              numeralDecimalScale: getTokenDecimalsWithFallback(
+                selectedToken && selectedToken.decimals,
+              ),
+            }}
+            disabled={!userHasPermission}
+          />
+          <div className={styles.tokenAmountSelect}>
+            <Select
+              label={MSG.token}
+              options={tokenOptions}
+              name="tokenAddress"
+              elementOnly
+              appearance={{ alignOptions: 'right', theme: 'grey' }}
+              disabled={!userHasPermission}
+            />
+          </div>
+          {values.tokenAddress === AddressZero && (
+            <div className={styles.tokenAmountUsd}>
+              <EthUsd
+                appearance={{ theme: 'grey', size: 'small' }}
+                value={
+                  /*
+                   * @NOTE Set value to 0 if amount is only the decimal point
+                   * Just entering the decimal point will pass it through to EthUsd
+                   * and that will try to fetch the balance for, which, obviously, will fail
+                   */
+                  values.amount && values.amount.length && values.amount !== '.'
+                    ? values.amount
+                    : 0
+                }
+              />
+            </div>
+          )}
+        </div>
+      </DialogSection>
+      <DialogSection>
+        <div className={styles.textAreaSection}>
+          <Textarea
+            appearance={{ resizable: 'vertical', colorSchema: 'grey' }}
+            label={MSG.annotation}
+            name="annotation"
+            maxLength={4000}
+            disabled={!userHasPermission}
+          />
+        </div>
+      </DialogSection>
+      {!userHasPermission && (
+        <DialogSection>
+          <span className={styles.noPermissionFromMessage}>
+            <FormattedMessage
+              {...MSG.noPermissionFrom}
+              values={{
+                firstRoleRequired: (
+                  <PermissionsLabel
+                    permission={ColonyRole.Funding}
+                    name={{ id: `role.${ColonyRole.Funding}` }}
+                  />
+                ),
+                secondRoleRequired: (
+                  <PermissionsLabel
+                    permission={ColonyRole.Administration}
+                    name={{ id: `role.${ColonyRole.Administration}` }}
+                  />
+                ),
+              }}
+            />
+          </span>
+        </DialogSection>
+      )}
+      <DialogSection appearance={{ align: 'right' }}>
+        <Button
+          appearance={{ theme: 'secondary', size: 'large' }}
+          onClick={cancel}
+          text={{ id: 'button.back' }}
+        />
+        <Button
+          appearance={{ theme: 'primary', size: 'large' }}
+          onClick={() => handleSubmit()}
+          text={{ id: 'button.confirm' }}
+          loading={isSubmitting}
+          disabled={!isValid}
```

Comment:
We need another check here: if the `OneTxPayment` extension is enabled in the colony. If it's not, we can't really create this transaction, so no point in allowing the user to even try.

But this can only be done after #2260 is merged.

You don't need to do anything, I just left a comment to remember this :slightly_smiling_face: 

I you really want to, you can leave a comment in the code as well...